src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -183,31 +183,33 @@ class adminLogs extends viewModelBase {
     }
     
     filterLogEntries(fromFilterChange: boolean) {
-        const searchText = this.filter().toLocaleLowerCase();
-        const errorsOnly = this.onlyErrors();
-
         if (fromFilterChange) {
             this.listController().reset();
         }
 
+        const filterFunction = this.getFilterFunction();
+
+        const filteredItems = fromFilterChange ?
+            this.allData.filter(filterFunction) :
```

Comment:
filter only if filter function is not null

Reply:
done